src/myrtlespeech/data/preprocess.py

```diff
@@ -222,3 +225,75 @@ def __repr__(self) -> str:
             + f" n_feature_masks={self.n_feature_masks},"
             + f" n_time_masks={self.n_time_masks})"
         )
+
+
+class MFCCLegacy:
+    """Legacy MFCC implementation using ``python_speech_features``.
+
+    This is required for backwards compatibility: **new users should not use
+    this class.** This class has the same API as
+    :py:class`torchaudio.transforms.MFCC`.
+
+    Args:
+        n_mfcc: Number of MFCC coefficients.
+
+        melkwargs: Dict containing mel-spectogram kwargs:
+
+            win_length:
+                Window length (in samples not ms.)
+
+            hop_length:
+                Stride length (in samples not ms.)
+
+        sample_rate: audio sample rate.
+    """
+
+    def __init__(self, n_mfcc, melkwargs, sample_rate=16000):
+        win_length = melkwargs.get("win_length")
```

Comment:
What's the purpose of using `get` here as opposed to indexing into the dictionary? If `win_length` isn't defined in the dictionary then `win_length = None` and `self.winlen = win_length / sample_rate` below will fail.

Reply:
good point - updated to index.